azure-functions-powershell-library/BundledBindings/OrchestrationTriggerBinding.cs

```diff
@@ -0,0 +1,26 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.Azure.Functions.PowerShellWorker;
+using System.Management.Automation.Language;
+
+namespace AzureFunctions.PowerShell.SDK.BundledBindings
+{
+    public class OrchestrationTriggerBinding : IInputBinding
+    {
+        public override string BindingAttributeName => "OrchestrationTrigger";
+
+        public override string BindingType => "orchestrationTrigger";
+
+        public override BindingInformation ExtractBinding(AttributeAst attribute, ParameterAst parameter)
+        {
+            BindingInformation bindingInformation = new BindingInformation();
```

Comment:
It's used in some of the bindings that implement IInputBinding, and it's an override constructor. It will always get passed, it's up to the binding whether it needs to use it or not